include/misc/byteorder.h

```diff
@@ -21,6 +21,14 @@
 				   (((x) >> 8) & 0xff00) | \
 				   (((x) & 0xff00) << 8) | \
 				   (((x) & 0xff) << 24)))
+#define __bswap_64(x) ((u64_t) ((((x) >> 56) & 0xff) | \
```

Comment:
ok but why is this patch part of lwm2m patchset? You probably want to send this patch in another PR.